script/pre-push.sh

```diff
@@ -10,26 +10,6 @@ then
   exit 1
 fi
 
-# echo "\033[34mRunning SCSS Lint\033[0m"
-# RUN_CHECK_CMD='bundle exec scss-lint app/assets/stylesheets/'
-# RUN_TESTS_OUTPUT=`${RUN_CHECK_CMD}`
-
-# if [ $? -eq 1 ]
-# then
-#   echo "\033[34mCan't commit! You have scss lint offences!!!\033[0m"
-#   exit 1
-# fi
-
-echo "\033[34mRunning eslint\033[0m"
-RUN_CHECK_CMD='yarn run lint'
-RUN_TESTS_OUTPUT=`${RUN_CHECK_CMD}`
-
-if [ $? -eq 1 ]
-then
-  echo "\033[34mCan't commit! You have eslint problems!!!\033[0m"
-  exit 1
-fi
-
 echo "\033[34mRunning Rubocop\033[0m"
```

Comment:
We can also add brakeman in this script

```
echo "\033[34mRunning Brakeman\033[0m"
RUN_CHECK_CMD='bundle exec brakeman --exit-on-error --quiet'
RUN_TESTS_OUTPUT=`${RUN_CHECK_CMD}`

if [ $? -ne 0 ]
then
  echo "\033[34mCan't commit! You have security warnings!!!\033[0m"
  exit 1
fi

echo "\033[34mAll checks passed. You didn't break anything. Congrats!\n\033[0m"
exit 0
```

Reply:
Sounds fine to me but we should do that in another PR so this actually gets merged 👀 wanna add that issue?